README.md

```diff
@@ -1,6 +1,112 @@
-# LBH Base Listener
+# MTFH Reporting Data Listener
+
+This listener implements an AWS Lambda Function that recieves messages when any updates are made to the TenureInformationApi and sends the updates to a Kafka queue. 
+
+Here is the process on how the data is saved to Kafka:
+1. When one of the events in [this switch statement][usecase-factory] is raised then this listener is triggered
+2. The listener calls the [TenureInformationApi][tenure-api-github] using a Shared Nuget Package to ensure that the Tenure sent through in the message exists and to get the details of the tenure. [The ReadME][Api-Gateway] for this package explains how the listener calls the TenureInformationApi.
```

Comment:
Hey Humairaa - as this is the README for the whole listener, can we make the descsription generic and not specific to Tenure information API? We can have a separate section that will show what use cases are currently implemeneted (e.g. Tenure updated)

Reply:
Sure, I can make that change